vignettes/customizing-dccvalidator.Rmd

```diff
@@ -124,11 +124,26 @@ To install the dccvalidator instead of forking the repository:
 * `teams`: The team(s) a user must be a member of in order to use the app. If
   the user is not in any of the teams, they will see a message telling them they
   must be added to one of the teams.
+* `include_biospecimen_type`: `TRUE` to include specific biospecimen types
+  within each species, else `FALSE`. This will dictate whether the Specimen Type
+  panel appears in the validator tab.
 * `templates` (including `manifest_template`, `individual_templates`,
   `biospecimen_templates`, and `assay_templates`): Location of templates to
-  use for validation. These may be a Synapse synID or the id of a Synapse-registered JSON schema. For templates accessed via synID, these should be either .xlsx or .csv files, where the
-  column names reflect the required columns in the template. If the template is
-  an excel file with multiple sheets, the first sheet will be used. For templates accessed via JSON schema id, these should have a `properties` object listing all column names (a.k.a. keys) that should be present in the metadata.
+  use for validation. These may be a Synapse synID or the id of a
+  Synapse-registered JSON schema. For templates accessed via synID, these should
+  be either .xlsx or .csv files, where the column names reflect the required
+  columns in the template. If the template is an excel file with multiple
+  sheets, the first sheet will be used. For templates accessed via JSON schema
+  id, these should have a `properties` object listing all column names (a.k.a.
+  keys) that should be present in the metadata. The `manifest_template` should
+  have only one template. `individual_templates` and `biospecimen_templates`
+  should be broken up by species (the species must be included in the
```

Comment:
I suggest revising the language to clarify "broken up by species": 
* `templates` (including `manifest_template`,......
     `individual_templates:`
                `<name species:> templates must be species-specific and the species must be named in a nested list under        `individual_templates`. 

It is difficult to capture the nested/indented relationship in this comment! Maybe instead of being explicit about the nested format, you could link to the appropriate line in the example config. Same thought goes for `biospecimen_templates`.

Reply:
This paragraph seemed cumbersome to me, as well! Thanks for the suggestion on how to make it better. I'll update it.

Reply:
Updated.